.eslintrc.json

```diff
@@ -203,7 +203,6 @@
                 "indentLogicalExpressions": true
             }
         ],
-        "react/jsx-no-bind": "error",
```

Comment:
The rule sounds for me something important in case of performance: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md

So not sure if I would remove it. I think I would go with keep it and add ignore where really required.

Reply:
i removed the rule because it makes list rendering a pain as you need to extract a component for the list item. in this case, i think the rule makes the code even slower because we need to instantiate a whole component for each item instead of just an arrow function 🙈 

but yeah, i guess there are cases where it improves performance too. i will readd it.

Reply:
> we need to instantiate a whole component for each item instead of just an arrow function

I see, but theoretically the component is initialized only once. And the function is created for every render so it also can in this case improve the performance. I'm just not sure how much mobx maybe would effect here things and maybe avoid rerender here something. 
Atleast we also created in this cases also for [Select Option](https://github.com/sulu/sulu/blob/2.5/src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js) own component to avoid this kind of problems. So I personally would go the save way of introducing also here a `MenuOption` or what the best name is here.